app/serializers/serializer_post_processing.rb

```diff
@@ -0,0 +1,18 @@
+# TODO: docs
+module SerializerPostProcessing
+  # TODO: docs
+  def serializable_object(options={})
+    super.tap {|hash|
+      modify_serializable_object(hash)
+    }
+  end
+
+  # TODO: docs
+  def modify_serializable_object(hash)
+    # TODO: having this method as an abstract overridable method is not very good,
```

Comment:
OK, I think I understand that better now. And I agree, we should rely on internal API as little as possible. And where you do, it's good when you document it.

Also as you mention it here: I tried to upgrade AMS in #7500, and "everything" broke when I used 0.10. I didn't have the time to have a closer look, so I just upgraded to the latest 0.9 version. But since you did more with serializers, you maybe understand the problems better than I and could have a look at the upgrade to 0.10?